spec/motion-spec/matchers/satisfy_spec.rb

```diff
@@ -0,0 +1,18 @@
+# -*- encoding : utf-8 -*-
+describe "Matcher::Satisfy" do
+  it 'satisfy passes when the block returns true' do
+    expect(1).to satisfy{|v| v == 1}
+  end
+
+  it 'satisfy fails when the block returns false' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.